### PR TITLE
Fix cat.health request and response

### PR DESCRIPTION
**Pull request by stevejgordon**

### CODE REVIEW

File: output/typescript/types.ts [hunk 1]
```diff
@@ -1017,7 +1017,7 @@ export type CatFielddataResponse = CatFielddataRecord[]
 
 export interface CatHealthRecord {
   cluster: string
-  epoch: string
+  epoch?: string
```
Comment (philkra) [inline output/typescript/types.ts:1020]: please use type `EpochMillis`

**Review by philkra — CHANGES_REQUESTED** [review]

File: output/typescript/types.ts (hunk 1 above)
Comment (stevejgordon) [reply output/typescript/types.ts:1020]: Done 😄

**Comment by stevejgordon** [comment]
Just untangling these conflicts!

**Comment by stevejgordon** [comment]
Oh! I see now, @philkra - You stole this one from my assigned APIs and it's already fixed up! 😄

**Pull request closed without merging**